nipst/nipst.go

```diff
@@ -28,6 +28,13 @@ type PostProverClient interface {
 	// the amortized computational complexity can be made arbitrarily small.
 	execute(id []byte, challenge []byte, timeout time.Duration) (proof *types.PostProof, err error)
 
+	// set the needed params for setting up post commitment in the specified logical drive and with
+	// requested commitment size
+	SetPostParams(logicalDrive string, commitmentSize uint64)
```

Comment:
Method doc should start with `SetPostParams`. Also, you can just call it `SetParams`.